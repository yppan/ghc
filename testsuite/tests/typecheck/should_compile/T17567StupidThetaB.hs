{-# LANGUAGE QuantifiedConstraints, DatatypeContexts, TypeFamilies #-}
-- NB: -XNoPolyKinds, to get defaulting.

module T17567StupidThetaB where

data (forall a. a b ~ a c) => T b c
