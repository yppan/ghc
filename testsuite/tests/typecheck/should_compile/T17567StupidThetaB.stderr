
T17567StupidThetaB.hs:1:37: warning:
    -XDatatypeContexts is deprecated: It was widely considered a misfeature, and has been removed from the Haskell language.
